packages/odyssey-storybook/src/components/odyssey-labs/FileUpload/FileUpload.stories.tsx

```diff
@@ -0,0 +1,100 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { FileUpload, fileUploadVariants } from "@okta/odyssey-react-mui/labs";
+import { Meta, StoryObj } from "@storybook/react";
+// import { userEvent, within } from "@storybook/testing-library";
+// import { expect } from "@storybook/jest";
+
+import { MuiThemeDecorator } from "../../../../.storybook/components";
+import { fieldComponentPropsMetaData } from "../../../fieldComponentPropsMetaData";
+// import { axeRun } from "../../../axe-util";
+
+const storybookMeta: Meta<typeof FileUpload> = {
+  title: "Labs Components/FileUpload",
+  component: FileUpload,
+  argTypes: {
+    type: {
+      control: "boolean",
+      description: "If `multiple`, the user can upload multiple files",
+      table: {
+        type: {
+          summary: "boolean",
+        },
+      },
+    },
+    hint: fieldComponentPropsMetaData.hint,
+    id: fieldComponentPropsMetaData.id,
+    isDisabled: fieldComponentPropsMetaData.isDisabled,
+    label: {
+      control: "text",
+      description: "The label text for the FileUpload",
+      table: {
+        type: {
+          summary: "string",
+        },
+      },
+      type: {
+        required: true,
+        name: "string",
+      },
+    },
+    onChange: {
+      control: null,
+      description: "Callback fired when a file is uploaded or deleted",
+      table: {
+        type: {
+          summary: "(files: File[]) => void",
+        },
+      },
+      type: {
+        name: "string",
+        required: true,
+      },
+    },
+    variant: {
+      options: fileUploadVariants,
+      control: { type: "radio" },
+      description: "The type of FileUpload",
+      table: {
+        type: {
+          required: true,
+          summary: fileUploadVariants.join(" | "),
```

Comment:
It can take an array, yes. But, this just looks slightly nicer IMHO. 
as an array:
<img width="316" alt="image" src="https://github.com/okta/odyssey/assets/148223016/e8b1c621-7133-4978-bfe1-bc70a32335f8">
vs
with join:
<img width="199" alt="image" src="https://github.com/okta/odyssey/assets/148223016/6efbb078-4772-4d73-854b-1cfb2fa01abc">

Reply:
I actually like the join one because it's easier to copy-paste. You can double-click in the box and take the whole thing, but with the commas, it highlights the whole line.

Reply:
Cool. I'll keep the `.join` then.